src/triage/component/postmodeling/contrast/model_group_evaluator.py

```diff
@@ -0,0 +1,889 @@
+"""
```

Comment:
This module looks nearly identical to the previous one.

1. Some elements, such as the docstring, appear to be truly, and erroneously, duplicated
2. My comments will likely be identical
3. Is there no way to avoid this much code repetition?